src/fat/bios_param.rs

```diff
@@ -48,13 +48,13 @@ impl BIOSParam {
     }
 
     /// Extract the BIOS Parameter Block (BPB) from the FAT filesystem image.
-    pub fn from_file<P: AsRef<Path>>(p: P)
+    pub fn from_file<P: AsRef<Path>>(p: P, o: usize)
```

Comment:
The arg names should be full words. That way you can tell what they are just from the function signature.